jsk_arc2017_baxter/euslisp/lib/util.l

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env roseus
+
+(require "package://jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/util.l")
+
+(ros::load-ros-manifest "jsk_gui_msgs")
+(ros::load-ros-manifest "jsk_recognition_msgs")
+
+(defun opposite-arm (arm) (if (eq arm :larm) :rarm :larm))
```

Comment:
@wkentaro 
> If you in package of jsk_arc2017_baxter, I mean by (in-package "jsk_arc2017_baxter), you can just type opposite-arm.

How do you acheive to use `(in-package "JSK_ARC2017_BAXTER")`? In command line, I checked it works, but it does not work in script file...